config.c

```diff
@@ -253,6 +325,35 @@ void read_FTLconf(void)
 		fclose(fp);
 }
 
+void getpath(FILE* fp, const char *option, const char *defaultloc, char **pointer)
```

Comment:
Could this function get some documentation (what it does, what the parameters mean, etc)?